npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater_spec.rb

```diff
@@ -2518,7 +2518,7 @@
                 to eq("git+ssh://git@github.com/jonschlinkert/is-number.git#"\
                       "0c6b15a88bc10cd47f67a09506399dfc9ddc075d")
               expect(parsed_package_lock["dependencies"]["is-number"]["from"]).
-                to eq("is-number@github:jonschlinkert/is-number#semver:^4.0.0")
+                to eq("is-number@jonschlinkert/is-number#semver:^4.0.0")
```

Comment:
Seems like npm fixed an issue here, this new `from` now mirrors the `package.json` dependency requirement https://github.com/dependabot/dependabot-core/blob/e5faa799769de92975587c8bb2b65c0c374c2f6d/npm_and_yarn/spec/fixtures/projects/npm7/github_dependency_semver_modern/package.json#L20